cmd/helm/install.go

```diff
@@ -273,6 +273,14 @@ func (i *installCmd) run() error {
 // Merges source and destination map, preferring values from the source map
 func mergeValues(dest map[string]interface{}, src map[string]interface{}) map[string]interface{} {
 	for k, v := range src {
+		// If the new value is literally the string "null", remove the value's key.
+		// This allows helm's multiple value sources (value files or --set) to
+		// remove incompatible keys from the previous values.
+		if v == "null" {
```

Comment:
- [ ] Note that "null" is only one of [the values that the YAML specification projects to null](http://www.yaml.org/spec/1.2/spec.html#id2803362).  
  There is also "Null," "NULL," and "~."